pkg/controllers/job/plugins/svc/svc.go

```diff
@@ -168,6 +168,14 @@ func (sp *servicePlugin) OnJobDelete(job *batch.Job) error {
 	}
 	delete(job.Status.ControlledResources, "plugin-"+sp.Name())
 
+	if !sp.disableNetworkPolicy {
+		if err := sp.Clientset.KubeClients.NetworkingV1().NetworkPolicies(job.Namespace).Delete(context.TODO(), job.Name, metav1.DeleteOptions{}); err != nil {
+			if !apierrors.IsNotFound(err) {
+				klog.Errorf("Failed to delete Network policy of Job %v/%v: %v", job.Namespace, job.Name, err)
+				return err
```

Comment:
what will happen if failed to delete network policy?

Reply:
network policy will left and controller print error log.  No impact to job finish process.
When job deleted, this network policy also be deleted by GC.